storybook/src/components/Button/Button.stories.tsx

```diff
@@ -35,24 +44,39 @@ export const Primary: Story = {}
 
 export const Secondary: Story = {
   args: {
+    children: 'Annuleren',
     variant: 'secondary',
   },
 }
 
 export const Tertiary: Story = {
   args: {
+    children: 'Terug',
     variant: 'tertiary',
   },
 }
 
 export const WithIcon: Story = {
   args: {
-    children: ['Button label', <Icon key="icon" svg={ShareIcon} size="level-5" />],
+    children: 'Bel 14 020',
```

Comment:
This isn't the best example, because this should be a link, not a button... How about a close icon with the label 'Sluiten'?

Reply:
Someone keeps tinkering with my PR's 😬. I changed it until it happens again.

Reply:
Sorry, thought I was helping.

Reply:
I'm a big fan of you doing the documenting, but perhaps it's more convenient to do it in a docs specific PR. To prevent the second reviewer requesting changes the first reviewer is doing.

Reply:
Yes, let’s talk a bit about this next Wednesday.